src/SW_Domain.c

```diff
@@ -210,9 +258,22 @@ void SW_DOM_read(SW_DOMAIN* SW_Domain, LOG_INFO* LogInfo) {
                     SW_Domain->SW_SpinUp.spinup = swTRUE;
                 }
                 break;
-            case 11: // Spinup Seed
+            case 16: // Spinup Seed
                 SW_Domain->SW_SpinUp.rng_seed = atoi( value );
                 break;
+            case 17: // Spinup Mode
```

Comment:
Just noticed there are 2 cases that read in the spinup mode. Not sure how it happened but I don't think we need both

Reply:
Thanks for catching that, @niteflyunicorns! It looks I made this mistake with commit https://github.com/DrylandEcology/SOILWAT2/commit/c72938acfb45288d97281acedd9bf7591e35bd66 when I improperly resolved merge conflicts when pulling in release/devel_v8.0.0 into feature_spinup
-> I fixed this with 864606a83c2ddcd9bb73fa1368ee537d346283fb